test_verbosefilter.py

```diff
@@ -0,0 +1,108 @@
+# -*- python -*-
+
+"""
+Unit tests for VerboseFilter, a logging helper in adept_openstack.py
+"""
+
+import logging
+import sys
+from mock import Mock
+from unittest2 import TestCase, main
+
+sys.modules['virtualenv'] = Mock()      # Needed for Ed's Fedora system
```

Comment:
Yep, this is fine.  It's documented as a dependency, but  none of what you're testing here goes anywhere needing it, so ok for everyone, not just you :smile: